### ISSUE TITLE
already in write mode error!

### ISSUE THREAD

**Issue by xion87** [kangarko/ChatControl-Pro#871]
after purging the file and mysql table file i still have this error.
The error happened after the player joined on same hub on 3° time

```
------------------------------------[ 01.08.2018 12:46:45 ]-----------------------------------
ChatControl 7.9.4 encountered a coM7
Running CraftBukkit 1.8.8-R0.1-SNAPSHOT and Java 1.8.0_161
Plugins: LuckPerms v4.2.17, WorldEdit v6.1.8-SNAPSHOT;cd4729f, Skript v2.2-SNAPSHOT, LampControl v3.0, PlaceholderAPI v2.8.2, VoidWorld v1.0, EasySetSpawn v2.0, MiniBoardGames v3.3.6, BuycraftX v10.6.0, BannerBoard v1.9.9, WorldGuard v5.9, SimpleTPA v2.1.12, DeluxePlayerOptions v1.6.2, MVdWPlaceholderAPI v2.5.1, Nicky v0.6.9, DeluxeMenus v1.10.1, SQLibrary v7.1, EasyWarp v3.8, ViaVersion v1.4.1, SuperDeliveryMan v2.3.2, Vault v1.5.6-b49, TitleManager v2.1.1, AnimatedNames v7.0.1b, PlayerPoints v2.1.4, ProtocolLib v3.6.5, LibsDisguises v8.6.8, MineSync vS_2.0.0, CustomPayloadFixer v1.5.1 X, GadgetsMenu v4.1.22, AuthMe v5.4.0-b1877, ChatControl v7.9.4, HolographicDisplays v2.2.6, ItemJoin v4.8-SNAPSHOT, BungeeNPC v1.3.6
----------------------------------------------------------------------------------------------

More Information: 
Player iTzGabryh_ is already in hub channel in write mode!
coM7 Report / Player iTzGabryh_ is already in hub channel in write mode!
	 at me.kangarko.chatcontrol.final. .LPt8.const(SourceFile:30)
	 at me.kangarko.chatcontrol.final. .O.const(SourceFile:103)
	 at me.kangarko.chatcontrol.final. .protected.setPlayer(SourceFile:133)
	 at me.kangarko.chatcontrol.final. .protected.joinPlayer(SourceFile:110)
	 at me.kangarko.chatcontrol.protected.const(SourceFile:282)
	 at me.kangarko.chatcontrol.protected.native(SourceFile:191)
	 at me.kangarko.chatcontrol.protected.<init>(SourceFile:167)
	 at me.kangarko.chatcontrol.ChatControl.const(SourceFile:507)
	 at me.kangarko.chatcontrol.ChatControl.const(SourceFile:500)
	 at me.kangarko.chatcontrol.final. .aT.const(SourceFile:154)
	 at me.kangarko.chatcontrol.final. .aT.const(SourceFile:53)
	 at me.kangarko.chatcontrol.final. .aT.const(SourceFile:35)
	 at me.kangarko.chatcontrol.final. .LPT1.execute(SourceFile:26)
	 at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74)
	 at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
	 at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517)
	 at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:499)
```

On mysql it look like this
![image](https://user-images.githubusercontent.com/3611852/43519553-31dd789a-9590-11e8-8ba8-a026600b69ba.png)

**Comment by xion87** [comment]
I forgot that i have added the following perms, the channel is "hub"
chatcontrol.channel.autojoin.hub.read
chatcontrol.channel.autojoin.hub.write
chatcontrol.channel.hub.join.read
chatcontrol.channel.hub.join.write

**Comment by kangarko** [comment]
Does it still throw on 7.9.9?

**Comment by kangarko** [comment]
Test 7.9.10 and repost a new stack trace. Also, when does it happen?

**Comment by xion87** [comment]
i still not tested, i gonna upgrade to 7.9.10 and see

**Comment by kangarko** [comment]
Should be fixed as per 7.9.10 or higher, closing.
